API.md

```
@@ -200,3 +200,40 @@ Allows players to offer or accept a draw agreement in PvP mode.
       "game_status": "draw_agreement" // Only present if action was "accept"
     }
     ```
+
+---
+
+## 9. Check Username Availability
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)